src/common/Testimonial/TestimonialCard.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { format } from 'date-fns';
+import * as allLocales from 'date-fns/locale';
+import { email2Slug } from 'common/services/string';
+
+const TestimonialCard = ({ home, quote, title, name, avatarUrl, category, created_at, email }) => {
+  const [formattedDate] = useState(() => {
+    const locale = navigator.language.split('-').join('');
+    const dnsLocale = allLocales[locale] ?? allLocales.enUS;
+
+    return format(new Date(created_at), 'MMM dd, yyyy', {
+      locale: dnsLocale
+    });
+  });
+
+  const getHostName = () => {
+    var url = window.location.href;
+    var arr = url.split('/');
```

Comment:
Why do you need it?

window.location.hostname should give you host name.

Reply:
we are using same in like play so i took it from there